logging/logrus/option.go

```diff
@@ -82,7 +82,7 @@ func WithTraceHookErrorSpanLevel(level logrus.Level) Option {
 	})
 }
 
-func WithRecordStackTraceInSpan(recordStackTraceInSpan bool) Option {
+func WithRecordStackTraceInSpan() Option {
```

Comment:
It feels like we still need to keep this parameter, and we need to have a mechanism to turn off stacktrace

Reply:
We can change it to this: `cfg.traceHookConfig.recordStackTraceInSpan = recordStackTraceInSpan`

Reply:
Because the default value is true 

<img width="710" alt="image" src="https://user-images.githubusercontent.com/19374680/174006333-bf720b50-127c-4ea9-8c22-93e66e668f82.png">